### PR TITLE
chore: update repository transfer in code

### LINKED ISSUE #95: Update repo transfer in code

### PR DESCRIPTION
**Pull request by mightyiam**
Closes #95.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-[![Build Status](https://travis-ci.org/mightyiam/eslint-config-standard-with-typescript.svg?branch=master)](https://travis-ci.org/mightyiam/eslint-config-standard-with-typescript)
+[![Build Status](https://travis-ci.org/standard/eslint-config-standard-with-typescript.svg?branch=master)](https://travis-ci.org/standard/eslint-config-standard-with-typescript)
```
Comment (mightyiam) [inline readme.md]: Why doesn't this image load?

File: readme.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
-[![Greenkeeper badge](https://badges.greenkeeper.io/mightyiam/eslint-config-standard-with-typescript.svg)](https://greenkeeper.io/)
-[![Releases](https://coderelease.io/badge/mightyiam/eslint-config-standard-with-typescript)](https://coderelease.io/github/repository/mightyiam/eslint-config-standard-with-typescript)
+[![Greenkeeper badge](https://badges.greenkeeper.io/standard/eslint-config-standard-with-typescript.svg)](https://greenkeeper.io/)
```
Comment (mightyiam) [inline readme.md]: I've requested installation of Greenkeeper from the admin of this org. Perhaps installation this image would load?

File: readme.md (hunk 1 above)
Comment (feross) [reply readme.md]: Loads for me! Says 'Build Failing'. It might be cached by your browser.

File: readme.md (hunk 2 above)
Comment (feross) [reply readme.md]: We already have Greenkeeper enabled on all repos in the org. It's strange that you needed to send a request. I logged into Greenkeeper and confirmed that they were just waiting for this PR to be merged: https://github.com/standard/eslint-config-standard-with-typescript/pull/91 which I have already done :)

**Pull request merged**